src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const copyRobot = {};
+
+  for (const key in robot) {
+    if (copyRobot[robot[key]]) {
+      return null;
+    }
```

Comment:
Unfortunately, this kind of check is even less reliable than the previous one.
It works with this particular task, but may fail in different cases.

The best way to check if object has certain property is by using `hasOwnProperty` method.